pkg/systemd/timer-dialog.jsx

```diff
@@ -154,31 +155,30 @@ const CreateTimerDialogBody = ({ owner }) => {
             {dialogError && <ModalError dialogError={_("Timer creation failed")} dialogErrorDetail={dialogError} />}
             <Form isHorizontal onSubmit={onSubmit}>
                 <FormGroup label={_("Name")}
-                           fieldId="servicename"
-                           validated={submitted && validationFailed.name ? "error" : "default"}
-                           helperTextInvalid={!name.trim().length ? _("This field cannot be empty") : _("Only alphabets, numbers, : , _ , . , @ , - are allowed")}>
+                           fieldId="servicename">
                     <TextInput id='servicename'
                                value={name}
                                validated={submitted && validationFailed.name ? "error" : "default"}
                                onChange={setName} />
+                    <FormHelper fieldId="servicename"
+                                helperTextInvalid={submitted && validationFailed.name && (!name.trim().length ? _("This field cannot be empty") : _("Only alphabets, numbers, : , _ , . , @ , - are allowed"))} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18641-20230415-073133-3760a938-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)